build-system/tasks/bundle-size/index.js

```diff
@@ -117,6 +115,7 @@ async function storeBundleSize() {
   const commitHash = gitCommitHash();
   log('Storing bundle sizes for commit', cyan(shortSha(commitHash)) + '...');
   try {
+    const requestPost = util.promisify(require('request').post); // Lazy-required to speed up task loading.
```

Comment:
Could this be hoisted to right under `const commitHash = ...` so it's not duplicated in the file?

Reply:
From my reading of the code, hoisting it there won't change anything, right? (This file calls only of the outer functions at a time, so the `require` will only be called once.)

Reply:
Sent out #33376.